ios/authentication.md

```diff
@@ -395,129 +468,174 @@ when configuring authentication using the AWS Amplify CLI.
-	     func application(
-	         _ application: UIApplication,
-	             didFinishLaunchingWithOptions launchOptions:
-	                 [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+Choose **Google** and then provide your Google **Client ID**.
 
-	          return AWSMobileClient.sharedInstance().interceptApplication(
-	              application, didFinishLaunchingWithOptions:
-	              launchOptions)
-	     }
+When configuration for Facebook sign-in is complete, the CLI displays a message confirming that you have configured local CLI metadata for this category. Run the following to update your changes in the cloud:
 
-	     // Other functions in AppDelegate . . .
+```terminal
+$ amplify push
+```
 
-	   }
-	```
+You can now [configure Google in your mobile app](./authentication#google-login-in-your-mobile-app).
+
+Note that the CLI allows you to select more than one identity provider for your app. You can also run `amplify auth update` to add an identity provider to an existing auth configuration.
+
+### Facebook Login in Your Mobile App
 
-2. Make sure you have a `UINavigationController` in your app to use the sign-in UI. The sign-in UI uses the `UINavigationController` as an anchor to perform all the transitions. Learn more about using [UINavigationController](https://medium.com/whoknows-swift/swift-the-hierarchy-of-uinavigationcontroller-programmatically-91631990f495).
-
-3. Implement your sign-in UI by calling the library provided by the SDK.
-
-	```swift
-	import UIKit
-	import AWSAuthCore
-	import AWSAuthUI
-
-	class SampleViewController: UIViewController {
-
-	 override func viewDidLoad() {
-
-	     super.viewDidLoad()
-
-	     // Call the showSignIn method from your `viewDidLoad` method
-	     // The showSignIn() method will check if the user is logged in,
-	     // and if the user is not logged in, it will present a sign-in UI using the navigation controller the view is part of.
-	     showSignIn()
-	 }
-
-	 func showSignIn() {
-	     if !AWSSignInManager.sharedInstance().isLoggedIn {
-	        AWSAuthUIViewController
-	          .presentViewController(with: self.navigationController!,
-	               configuration: nil,
-	               completionHandler: { (provider: AWSSignInProvider, error: Error?) in
-	                  if error != nil {
-	                      print("Error occurred: \(String(describing: error))")
-	                  } else {
-	                      // Sign in successful.
-	                  }
-	               })
-	     }
-	 }
-	}
+1. Add or update your AWS backend configuration file to incorporate your new sign-in. For details, see the last steps in the [Get Started: Set Up Your Backend](./add-aws-mobile-sdk-basic-setup) section.
+
+2. Add the following dependencies in your project's `Podfile`.
+
+	```ruby
+	platform :ios, '9.0'
+	  target 'YOUR-APP-NAME' do
+	    use_frameworks!
+	    pod 'AWSMobileClient', '~> 2.6.33'
+	    pod 'AWSFacebookSignIn', '~> 2.6.33'
```

Comment:
We can callout in comments that `AWSFacebookSignIn`, `AWSAuthUI` is needed only for using drop in UI, else can be skipped.

Reply:
Done